tools/scripts/tr-archive/archive.js

```diff
@@ -0,0 +1,71 @@
+const fs = require('fs').promises;
+const marked = require("marked");
+const jsdom = require("jsdom");
```

Comment:
Not a big deal but you have mixed double quotes and single quotes 🙂

Reply:
@styfle thanks! I'll run the prettier on the file